include/RAJA/policy/sycl/forall.hpp

```diff
@@ -82,7 +82,8 @@ cl::sycl::range<1> getGridDim(size_t len, size_t block_size)
 ////////////////////////////////////////////////////////////////////////
 //
 
-template <typename Iterable, typename LoopBody, size_t BlockSize, bool Async>
+template <typename Iterable, typename LoopBody, size_t BlockSize, bool Async,
+          typename std::enable_if<std::is_trivially_copyable<LoopBody>{},bool>::type = true>
```

Comment:
This confused me for a bit, does it make sense to use ::value instead of {} for consistency?

Reply:
That would fit our conventions better.  Given that this is sycl code though, and it can only compile in C++17 mode anyway, it should probably be `std::is_trivially_copyable_v<>`.  That version removes the construction that we don't actually need.

Reply:
I've made the update